guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,56 +1,72 @@
+To achieve this, you create filters to exclude most of the errata after the {RHEL}{nbsp}8.3 release date, except a few that you need.
+Then, you enable dependency solving.
 
-ifeval::["{client-os-family}" == "Red Hat"]
-In some cases, excluding packages may make dependency solving impossible for DNF.
-For example, if you make a {RHEL} 8.3 repository with a few excluded packages, `dnf update` could fail.
-Instead of turning on dependency solving to resolve the issue, investigate the error from DNF and adjust the filters to stop excluding the missing dependency.
-If dependency solving is enabled, the repository may stray from {RHEL} 8.3 as discussed above.
-endif::[]
+In this situation, dependency solving might include more packages than expected.
+As a result, the client digresses from being a {RHEL}{nbsp}8.3 machine.
 
-If a Content View version is out of date, packages that are part of a newer erratum may depend on packages that are not within the Content View version.
-Before incrementally adding the erratum with dependency solving, consider publishing a new Content View version instead.
-Incrementally adding the erratum with dependency solving might pull in many new packages, which could alter the intended scope of the Content View version.
+If you do not need the extra errata and packages, do not configure Content View filtering.
+Instead, enable and use the {RHEL}{nbsp}8.3 repository on the *Content* > *Red Hat Repositories* page in the {ProjectWebUI}.
+====
+endif::[]
 
-If a single package is incrementally added to a Content View version, dependency solving may be needed to install that package.
-Keep in mind that the dependency solving recommendations above also apply to incremental updates.
+ifeval::["{client-os-family}" == "Red Hat"]
+[id="Excluding_packages_and_dependency_solving_with_DNF"]
+.Excluding packages sometimes makes dependency solving imposible for DNF
+====
+If you make a {RHEL}{nbsp}8.3 repository with a few excluded packages, `dnf update` can sometimes fail.
 
-If dependency solving must be used, consider using more Content Views with fewer repositories and combining them together using composite Content Views.
-Dependency solving time increases significantly with more repositories because every repository in a Content View is examined for dependencies.
+Do not enable dependency solving to resolve the problem.
+Instead, investigate the error from `dnf` and adjust the filters to stop excluding the missing dependency. 
 
-.Enabling Dependency Solving
-You can select to resolve package dependencies per Content View or have it enabled by default in *Settings > Content > "content view Dependency Solving Default"*.
-{Team} does not recommend enabling Content View dependency solving by default unless the pitfalls above are well-understood.
+Otherwise, dependency solving might cause the repository to digress from {RHEL} 8.3.
```

Comment:
```suggestion
Else, dependency solving might diverge the repository from {RHEL} 8.3.
```
We mostly use `Else`.
`Digress` is a new word for me. I am not aware if it is included in Style Guide. I would rather prefer `diverge`.